### DISCUSSION TITLE
zsh: command not found: glances

### DISCUSSION THREAD

**Discussion by Cheezik** [nicolargo/glances#2666] [General]
**Describe the bug**
ZSH shell unable to find glances(?)

**To Reproduce**
1. Install Glances (pip install --user glances)
2. Be using ZSH shell (or switch into it)
3. Error: "zsh: command not found: glances"

**Expected behavior**
Opening glances

**Screenshots**
![image](https://github.com/nicolargo/glances/assets/80171791/630bca43-8293-4664-aeea-c4b30e8dac76)
![image](https://github.com/nicolargo/glances/assets/80171791/14e06924-55a4-4fab-8966-4124c6d85f00)

**Environement (please complete the following information)**
 - Debian GNU/Linux 10 (buster)
 - Glances & psutil versions: `Glances v3.4.0.3 with PsUtil v5.9.8
 - How do you install Glances (Pypi package, script, package manager, source): `pip install --user glances`
 - Glances test (only available with Glances 3.1.7 or higher):
 - Shell: ZSH

**Additional context**
To make it clear, Glances work on Bash shell. i am making this just to notify Maintainers that this issue exists.

**Comment by lnlyssg** [comment]
Check your .bashrc file, you probably have a line like `export PATH="$PATH:$HOME/.local/bin"` that is missing from your .zshrc, add the line to .zshrc and you should be good to go.
